packages/ionic-angular/package.json

```diff
@@ -26,7 +26,7 @@
     "@phenomnomnominal/tsquery": "~5.0.1"
   },
   "peerDependencies": {
-    "@nx/devkit": "^19.0.0",
-    "@nx/angular": "^19.0.0"
+    "@nx/devkit": "20.0.0",
+    "@nx/angular": "20.0.0"
```

Comment:
Same for all peer dependencies as mentioned before.

Reply:
Damn! 🤣 will change everywhere in all package deps/peerDeps

Reply:
why linting target cannot do that automatically? Or it can? Do you know something about  that @edbzn ? 🤔

Reply:
done